packages/core/content-releases/shared/contracts/release-action.ts

```diff
@@ -0,0 +1,36 @@
+import type { Entity, Common } from '@strapi/types';
+import type { Release } from './release';
+import type { errors } from '@strapi/utils';
+
+interface ReleaseActionEntry {
+  id: Entity.ID;
+  [key: string]: unknown;
```

Comment:
is this supposed to be the attributes or....?

Reply:
Yes, but I think we can use Entity instead of define a new "ReleaseActionEntry" @markkaylor 

Because the definition of Entity is almost exactly this same, so why don't use it directly on `ReleaseAction`?

Reply:
Yes this is supposed to  be the attributes of any contentType. @Feranchz not sure I completely understood what you meant. I pushed an update with an idea though so lmk what y'all think.

Reply:
If this is meant to be a contentType it might better extending the Schema.ContentType type from `@strapi/types` alternatively, you could do `[key: string]: Attribute.Any` where `Attribute` is also imported from `@strapi/types`.